docker-compose.yml

```diff
@@ -45,13 +49,18 @@ services:
             # BEHAVIOR_INBOUNDTTL TODO
             - BEHAVIOR_CTXPROPAGATION=ctxclient,ctxserver,transport
             - BEHAVIOR_APACHETHRIFT=apachethriftclient,apachethriftserver
+            - BEHAVIOR_ONEWAY=client_oneway,server_oneway,transport_oneway,encoding
+            - BEHAVIOR_ONEWAYCTXPROPAGATION=client_oneway,server_oneway,transport_oneway
```

Comment:
changed to ONEWAY_CTXPROPAGATION
I can't use the hyphens since they correspond to the AXIS_* variables